data/movements/scripts/snow.lua

```diff
@@ -1,6 +1,6 @@
 function onStepOut(creature, item, position, fromPosition)
-	if creature:isPlayer() and creature:isInGhostMode() then
-		return true
+	if not creature:isPlayer() or creature:isInGhostMode() then
+		return false
```

Comment:
It seems like the return value of onStep* functions does not matter, but when it would, then you would be blocking moving out of a tile by returning `false` here. 🤔

IMHO it should still be returning `true` or not return anything at all. Just something to think about.

Reply:
I agree, there is no point to return false even if it "works".